app/pages/preview/contextCreator.test.js

```diff
@@ -0,0 +1,38 @@
+import manifest from './manifest.json';
+import { getContext } from './contextCreator';
+import { baseUrl } from '../../config';
+
+describe('order summary preview contextCreator', () => {
+  describe('getContext', () => {
+    it('should return the backLinkText', () => {
+      const context = getContext({ orderId: 'order-1', orderData: { description: 'Some order description' } });
```

Comment:
could this const be set at the top they're all the same

Reply:
👍 good shout. I'll make this change in the next PR